app/views/static/references.html.erb

```diff
@@ -9,30 +9,81 @@
+        <p class='static-page__text static-page__text--sinai'>Aleksidze, et al. = Aleksidze, Z., et al. <i>Catalogue of Georgian manuscripts discovered in 1975 at St. Catherine&#39;s Monastery on Mount Sinai</i>. Athens, 2005. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/1131665083&amp;sa=D&amp;source=editors&amp;ust=1683922505976172&amp;usg=AOvVaw3i-kfj6WhY44OytwlAWjP8"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Clark = <i>Checklist of Manuscripts in St. Catherine’s Monastery Mount Sinai</i>, ed. Kenneth W. Clark. Washington, D.C.: Library of Congress, 1952. [<%= link_to 'Library of Congress', "http://lcweb2.loc.gov/service/gdc/scd0001/2012/20120109002ch/20120109002ch.pdf", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Atiya = Atiya, Aziz Suryal. <i>The Arabic Manuscripts of Mount Sinai: A Hand-List of the Arabic Manuscripts and Scrolls Microfilmed at the Library of the Monastery of St. Catherine, Mount Sinai</i>. Johns Hopkins Press, 1955. [<%= link_to 'Internet Archive', 'https://archive.org/details/arabicmanuscript0000atiy_m0l2/mode/2up&amp;sa=D&amp;source=editors&amp;ust=1683922505976787&amp;usg=AOvVaw1wl_4qvUCtSOrny8cXdSzW"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Géhin 2006 = Géhin, Paul. Manuscrits sinaïtiques dispersés, I:  les fragments arabes et syriaques de Paris, OC 90 2006, 23-43. [<%= link_to 'WorldCat', "https://www.worldcat.org/title/-manuscrits-sinaitiques-disperses-i-les-fragments-syriaques-et-arabes-de-paris/oclc/784157423&referer=brief_results", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Balageorgos and Kriticou = Balageorgos and Kriticou. <i>Les manuscrits de musique byzantine: Sina&iuml;</i>. Athens, 2008. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/660172233&amp;sa=D&amp;source=editors&amp;ust=1683922505977243&amp;usg=AOvVaw1017NDAS229ZLuHvnOnc3s"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Géhin 2017 = Géhin, Paul. <i>Les manuscrits syriaques de parchemin du Sinaï et leurs</i> membra disjecta (CSCO 665, Subsidia 136; Leuven, 2017). [<%= link_to 'WorldCat', "https://www.worldcat.org/title/manuscrits-syriaques-de-parchemin-du-sinai-et-leurs-membra-disjecta/oclc/1000455293&referer=brief_results", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Brock 1995 = S.P. Brock. <i>Catalogue of Syriac Fragments (New Finds) in the Library of the Monastery of St Catherine, Mount Sinai</i>(Athens, 1995). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/catalogue-of-syriac-fragments-new-finds-in-the-library-of-the-monastery-of-saint-catherine-mount-sinai/oclc/780492566?referer%3Ddi%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505977880&amp;usg=AOvVaw3YjFUlR-yRhXfx2L9dpFxz"', class: 'si-link' %>]</p>
```

Comment:
Missing space: `Sinai</i>(Athens` -> `Sinai</i> (Athens`

Reply:
URL should be: https://www.worldcat.org/title/catalogue-of-syriac-fragments-new-finds-in-the-library-of-the-monastery-of-saint-catherine-mount-sinai/oclc/780492566